fluss-server/src/main/java/com/alibaba/fluss/server/coordinator/CoordinatorEventProcessor.java

```diff
@@ -884,6 +909,90 @@ private void processDeadTabletServer(DeadTabletServerEvent deadTabletServerEvent
         updateTabletServerMetadataCache(serverInfos, null, null, bucketsWithOfflineLeader);
     }
 
+    private AddServerTagResponse processAddServerTag(AddServerTagEvent event) {
```

Comment:
I have a question: Should we also update the ServerTag in `serverMetadataCache` and access the ServerTag by `serverMetadataCache`?